defaults/test_default.yaml

```diff
@@ -250,3 +250,5 @@ teardown_validators:
 
 kafka_backend: null
 kafka_connectors: []
+
+enable_force_gossip_topology_changes: false
```

Comment:
why `enable_force_gossip_topology_changes` and not `force_gossip_topology_changes`?

Reply:
renamed it